deflect/NetworkListener.h

```diff
@@ -55,14 +56,17 @@ class NetworkListener : public QTcpServer
 
 public:
     /** The default port number used for Stream connections. */
-    DEFLECT_API static const int defaultPortNumber_;
+    DEFLECT_API static const int defaultPortNumber;
```

Comment:
Honestly...What does it have to do with my original PR/where do we stop here? 

I'm happy fixing things/making the code better, but it's been a week now that I try to get this relatively simple PR merged.